cdap-notifications/src/main/java/co/cask/cdap/notifications/feeds/service/NotificationFeedStore.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.notifications.feeds.service;
+
+import co.cask.cdap.notifications.feeds.NotificationFeed;
+
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * Store for {@link NotificationFeed} objects.
+ */
+public interface NotificationFeedStore {
+
+  /**
+   * Creates a new Notification feed.
+   *
+   * @param feed {@link NotificationFeed} representing the feed.
+   * @return existing {@link NotificationFeed} if a feed with the same id already exists,
+   * null if no feed with the same id exists, and the feed was created successfully.
```

Comment:
Is this correct? Shouldn't the param by a String feedId? I'm having trouble parsing the @return clause...